package.json

```diff
@@ -7,7 +7,8 @@
     "preview": "nuxt preview",
     "postinstall": "nuxt prepare",
     "lint": "eslint --ext .ts,.js,.vue .",
-    "typecheck": "nuxi typecheck"
+    "typecheck": "nuxi typecheck",
```

Comment:
typo? `nuxt`

Reply:
typo

Reply:
Are we sure this is a typo? 🤔  I'm not familiar with Nuxt 3 that much yet but I've seen a lot of commands related to it use `nuxi` keyword. I checked docs and found this: https://nuxt.com/docs/api/commands/typecheck#nuxi-typecheck

Reply:
😮 oh! If it's right then sorry 🙈

Reply:
I see that nuxi is Nuxt3 CLI, so my bad... But also, since it's not in dependencies, shouldn't it be `npx nuxi typecheck`? I'm just trying to understand, as I never did anything in Nuxt3 🙈